src/main/java/ch/njol/skript/log/SkriptLogger.java

```diff
@@ -97,47 +91,54 @@ public static ParseLogHandler startParseLogHandler() {
 	 * @see FilteringLogHandler
 	 * @see RedirectingLogHandler
 	 */
-	public static <T extends LogHandler> T startLogHandler(final T h) {
-		handlers.add(h);
+	public static <T extends LogHandler> T startLogHandler(T h) {
+		getHandlers().add(h);
 		return h;
 	}
 	
-	static void removeHandler(final LogHandler h) {
+	static void removeHandler(LogHandler h) {
+		HandlerList handlers = getHandlers();
 		if (!handlers.contains(h))
 			return;
 		if (!h.equals(handlers.remove())) {
 			int i = 1;
 			while (!h.equals(handlers.remove()))
 				i++;
-			LOGGER.severe("[Skript] " + i + " log handler" + (i == 1 ? " was" : "s were") + " not stopped properly! (at " + getCaller() + ") [if you're a server admin and you see this message please file a bug report at https://github.com/bensku/skript/issues if there is not already one]");
+			LOGGER.severe("[Skript] " + i + " log handler" + (i == 1 ? " was" : "s were") + " not stopped properly!" +
+				" (at " + getCaller() + ") " +
+				"[if you're a server admin and you see this message please file a bug report at https://github.com/bensku/skript/issues if there is not already one]");
```

Comment:
GitHub redirects anyway, but, might as well change the link to use `SkriptLang/Skript`, while on at it.